README.md

```diff
@@ -1,5 +1,5 @@
 1. Replace `<your_account>` with your Github username in the link
-    - [DEMO LINK](https://<your_account>.github.io/js_gallery_DOM/)
+    - [DEMO LINK](https://vtroni.github.io/js_gallery_DOM/)
```

Comment:
Add this link to the PR description 
<img width="940" alt="image" src="https://github.com/user-attachments/assets/b404b9f8-009d-4840-830f-7d8dd18aed08">